portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,52 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+document.addEventListener('DOMContentLoaded',function(event){
+  function toggleButton() {
+    document.getElementById("upArrow").className = 'show'; 
+    document.getElementById("downArrow").className = 'show'; 
+    document.getElementById("but1").className = 'btn btn-dark btn-lg show'; 
+    document.getElementById("but2").className = 'btn btn-dark btn-lg show'; 
+}
+  // array with texts to type in typewriter
+  var dataText = [ "Hello! My name is Amita Gondi.", 
+                  "The whole covid-19 situation sucks. A lot.", 
+                  "But that gave me an opportunity to blow the dust off of the old travel albums. Metaphorically.", 
+                  "Thus, presenting", 
+                  "Amita's Travel Album Circa 2001"];
+  
+  // type one text in the typwriter
+  // will keep calling itself until text from array is done
+  function typeWriter(text, i, fnCallback) {
+    // check if text isn't finished yet
+    if (i < (text.length)) {
+      // add next character to h1
+     document.querySelector("h1").innerHTML = text.substring(0, i+1) +'<span aria-hidden="true"></span>';
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+      // wait for a while and call this function again for next character
+      setTimeout(function() {
+        typeWriter(text, i + 1, fnCallback)
+      }, 100);
+    }
+    // text finished, call callback if there is a callback function
+    else if (typeof fnCallback == 'function') {
+      // call callback after timeout
+      setTimeout(fnCallback, 700);
+    }
+  }
+  // start a typewriter animation for a text in the dataText array
+   function StartTextAnimation(i) {
```

Comment:
function names in javascript usually follows camelCase notation i.e. `startTextAnimation`

Reply:
Updated